docs/tutorials/getting_started.ipynb

```diff
@@ -31,1089 +31,160 @@
     "\n",
```

Comment:
<p>This text has clearly been <s>plagiarised</s> copied</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/21cmfast/tuesday/pull/28/discussion/#comment-2097792290'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>